bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractBaseImpl.java

```diff
@@ -91,6 +91,8 @@ public abstract class AbstractBaseImpl extends AbstractFormComponentImpl impleme
     protected String assistPriorityJcr;
     private AssistPriority assistPriority;
 
+    protected String contextPath = StringUtils.EMPTY;
```

Comment:
Let's move this to FormContainer itself, I don't see any use-case to use it elsewhere.